app/views/spotlight/translations/_browse_categories.html.erb

```diff
@@ -0,0 +1,67 @@
+<div role="tabpanel" class="tab-pane <%= 'active' if @tab == 'browse' %>" id="browse">
+  <%= bootstrap_form_for current_exhibit, url: spotlight.exhibit_translations_path(current_exhibit), layout: :horizontal do |f| %>
+    <% # Add hidden fields for the language and tab so the redirect knows how to come back here %>
+    <%= hidden_field_tag :language, @language %>
+    <%= hidden_field_tag :tab, 'browse' %>
+    <div class="row">
+      <div class="col-xs-4 text-right">
+        <strong>
+          <%= t :'spotlight.exhibits.translations.browse_categories.default_language_column_label' %>
+        </strong>
+      </div>
+      <div class="col-xs-7">
+        <strong>
+          <%= t :'spotlight.exhibits.translations.browse_categories.translation_column_label', language: t("locales.#{@language.downcase}")  %>
+        </strong>
+      </div>
+    </div>
+
+    <% current_exhibit.searches.each do |search| %>
+      <% title_translation = Translation.find_or_initialize_by(exhibit: current_exhibit, key: "#{search.slug}.title", locale: @language) %>
+      <%= f.fields_for :translations, title_translation do |translation_fields| %>
```

Comment:
I wonder if we can extract a partial for these translations (here and in other similar files)? Maybe there's less similarity than it appears on the surface, though.. something to think about.